pants.toml

```diff
@@ -14,14 +14,14 @@ backend_packages.add = [
   "pants.backend.python.typecheck.mypy",
   "pants.backend.python.mixed_interpreter_constraints",
   "internal_plugins.releases",
-  "toolchain.pants.auth",
-  "toolchain.pants.buildsense",
-  "toolchain.pants.common",
+#  "toolchain.pants.auth",
+#  "toolchain.pants.buildsense",
+#  "toolchain.pants.common",
```

Comment:
Yeah this is blocked by https://github.com/pantsbuild/pants/issues/11469, which Liam is working on. It's okay to temporarily turn these off. 

What we will really want to do is find a way to turn off only in the Py39 jobs, but that can wait till everything is green.